exercises/accumulate/README.md

```diff
@@ -25,6 +25,9 @@ Keep your hands off that collect/map/fmap/whatchamacallit functionality
 provided by your standard library!
 Solve this one yourself using other basic tools instead.
 
+Lisp specific: it's perfectly fine to use `MAPCAR` or the equivalent,
```

Comment:
This adds a line that was removed in https://github.com/exercism/rust/pull/342 so it should not be in this PR.

Reply:
well, the fact that it changed in #342 is irrelevant, the reason it should not be in this PR is because it is an exercise other than perfect-numbers, and this is supposed to be a PR about perfect-numbers.